e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -262,7 +262,7 @@ export function dashboardSaveButton() {
 }
 
 /**
- * @param {Object=} option
+ * @param {Object} option
```

Comment:
I don't know why making `Object` optional with `=` suffix breaks VS Code autocompletion here. I tested that using `[option]` also works.